src/pages/Results/Results.js

```diff
@@ -47,14 +48,20 @@ const Results = () => {
                     {Object.keys(checkboxes).map(key => (
                         <div className="filter-category">
                             <h3> {key} </h3>
-                            { checkboxes[key].names.map(item => (
+                            { checkboxes[key].names.map(item => ( key !== "Experience Level" ? 
                                 <div>
                                     <Checkbox name={item.name} checked={checkedItems[item.name]?.checked} onChange={event => handleChange(event, key, checkboxes[key].predicate)} />
                                     <label key={item.name}>
                                         {item.name}
                                     </label>
                                 </div>
-                            ))}
+                                :
+                                <div className='Experience-Level'>
+                                    <Checkbox name={item.name} checked={checkedItems[item.name]?.checked} onChange={event => handleChange(event, key, checkboxes[key].predicate)} />
+                                    <DifficultyExplanation difficulty={item.name} />
+                                </div>
+                                ))
+                            }
```

Comment:
I haven't tried this myself, but if you can, it'd be great to combine these as much as possible in terms of potentially only making L54-56 and L61 within the ternary operation. 

If you find that it doesn't work easily, feel free to dismiss!

Reply:
I'll have to dismiss this one! Not too sure how to implement it effectively.